src/components/admin/hocs/modals/ModalProvider.jsx

```diff
@@ -34,6 +35,11 @@ export class ModalProvider extends React.Component {
     };
   }
 
+  componentWillMount() {
+    /** Make the logger use this prettier display function */
```

Comment:
Comment formatting here should be `//` no?

Reply:
Yeah I guess consistency is good, I can change that :)